src/libcore/option.rs

```diff
@@ -193,6 +194,19 @@ impl<T> Option<T> {
         }
     }
 
+    #[rustc_const_unstable(feature = "const_option_match")]
+    #[cfg(not(bootstrap))]
+    #[must_use = "if you intended to assert that this has a value, consider `.unwrap()` instead"]
+    #[inline]
+    #[stable(feature = "rust1", since = "1.0.0")]
+    pub const fn is_some(&self) -> bool {
```

Comment:
The documentation is on the copy that will disappear instead of on this one, same with others in this file

Reply:
Done.